OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -485,7 +486,8 @@ public static void AddApplicationServices(this WebApplicationBuilder builder)
         services.AddEmailSender(
             builder.Environment.IsDevelopment(),
             mailConfig.SendGridKey,
-            builder => builder.Bind(configuration.GetSection(EmailOptions.SectionName)));
+            builder => builder.Bind(configuration.GetSection(EmailOptions.SectionName)))
+            .AddEmailRendererConfiguration(new EmailContentConfig(configuration.GetSection("Hosts")["BackendUrl"]));
```

Comment:
(Logic) `configuration.GetSection("Hosts")["BackendUrl"]` - here string keys are different from mentioned in `AuthCommonServiceExtensions.cs`. Is it intended to have same thing named differently?

Reply:
Sorry to have come back to this PR after a few months :) but had to do other email refactorings first. Yes the reason to create the config object is to unify the string keys because in their own projects they have own logic to be used that way and we wanted to reuse the keys without dublicating.